sympy/external/tests/test_numpy.py

```diff
@@ -138,9 +139,11 @@ def test_Matrix1():
 
 def test_Matrix2():
     m = Matrix([[x, x**2], [5, 2/x]])
-    assert (matrix(m.subs(x, 2)) == matrix([[2, 4], [5, 1]])).all()
+    with ignore_warnings(PendingDeprecationWarning):
+        assert (matrix(m.subs(x, 2)) == matrix([[2, 4], [5, 1]])).all()
```

Comment:
As far as I can tell all the tests involving `np.matrix` are just checking that it can be used with sympy. It doesn't look like sympy actually uses `np.matrix` for anything.